src/libs/tools/tests/testtool_umount.cpp

```diff
@@ -60,25 +60,27 @@ TEST (Umount, InvolvedRoot)
 
 TEST (Umount, InvolvedSpec)
 {
-	testUmount ("spec/is///a//./more/complex/../complicated///issue//.");
+	testUmount ("spec:/is///a//./more/complex/../complicated///issue//.");
 }
 TEST (Umount, InvolvedDir)
 {
-	testUmount ("dir/is///a//./more/complex/../complicated///issue//.");
+	testUmount ("dir:/is///a//./more/complex/../complicated///issue//.");
 }
 TEST (Umount, InvolvedUser)
 {
-	testUmount ("user/is///a//./more/complex/../complicated///issue//.");
+	testUmount ("user:/is///a//./more/complex/../complicated///issue//.");
 }
 TEST (Umount, InvolvedSystem)
 {
-	testUmount ("system/is///a//./more/complex/../complicated///issue//.");
+	testUmount ("system:/is///a//./more/complex/../complicated///issue//.");
 }
 TEST (Umount, InvolvedCascading)
 {
 	testUmount ("/is///a//./more/complex/../complicated///issue//.");
 }
 
+// TODO (kodebach): what is oldmount ??
+
 /**
  * @brief Test umount with compatibility umount-names
```

Comment:
@markus2330 What are these "old mount" tests below? Do we still need them or can I remove them?

Reply:
Here is the permalink: https://github.com/ElektraInitiative/libelektra/blob/3982b81ed209964ff598db8bedf97fa02c7320c5/src/libs/tools/tests/testtool_umount.cpp#L85-L139

Reply:
If we get rid of the fallback code (see permalink below), then we can also remove the tests. But maybe in a different PR?

https://github.com/ElektraInitiative/libelektra/blob/3982b81ed209964ff598db8bedf97fa02c7320c5/src/libs/elektra/kdb.c#L211-L249

Reply:
> But maybe in a different PR?

Exactly what I didn't want...

`testtool_umount` currently fails because of these "old mount" tests. I haven't found the cause yet, that's why I am asking, if removing them would be okay.

Reply:
Yes, it is okay. doc/decisions/bootstrap.md also needs to be adapted, then.

Reply:
Please check, that I adapted `doc/decisions/bootstrap.md` correctly.